sdk/python/lib/pulumi/asset.py

```diff
@@ -81,41 +82,44 @@ class Archive(object):
 
 @known_types.asset_archive
 class AssetArchive(Archive):
+    assets: Dict[Any, Union[Asset, Archive]]
```

Comment:
Should this be a `Dict[str, Union[...]]`? That's how we type it in the Node.js SDK.